highs/presolve/HPresolve.cpp

```diff
@@ -4523,6 +4538,100 @@ HPresolve::Result HPresolve::detectDominatedCol(
   return Result::kOk;
 }
 
+void HPresolve::dualBoundTightening(HighsPostsolveStack& postsolve_stack,
+                                    HighsInt col) {
+  // tighten bounds using dual arguments
+  // see section 4.4 "Dual fixing, substitution and bound strengthening",
+  // Achterberg et al., Presolve Reductions in Mixed Integer Programming,
+  // INFORMS Journal on Computing 32(2):473-506.
+  assert(!colDeleted[col]);
+
+  // only tighten bounds for integer-constrained variables
+  if (model->integrality_[col] == HighsVarType::kContinuous) return;
```

Comment:
I don't understand why this has to be restricted to integer columns. (I'm probably missing something instead of it being wrong)